File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        // Count of positive and negative integers
        int positive = 0;
        int negative = 0;

        // For each loop
        for (int n : nums) {
            // If integer biger than 1 increase positive
            if (n >= 1) {
                positive++;
            }
            // If integer smaller than 0 than increase negative
            if (n < 0) {
                negative++;
            }
        }

        // Return the maximum between positive and negative integers counts
        return Math.max(positive, negative);
    }
}

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        # Liner Sweep
        # max
        pos = 0
        neg = 0
        for n in nums:
            if n > 0:
                pos += 1
            if n < 0:
                neg += 1
        return max(pos, neg)
        